.env

```diff
@@ -3,8 +3,14 @@
 LLAMA_CPP_SERVER_PORT=8000
 LLAMA_CPP_HOST=llama-cpp-server
 LOGDETECTIVE_SERVER_PORT=8080
-MODEL_FILEPATH=/models/mistral-7b-instruct-v0.2.Q4_K_S.gguf
 # for some reason, fastapi cripples sys.path and some deps cannot be found
 PYTHONPATH=/src:/usr/local/lib64/python3.12/site-packages:/usr/lib64/python312.zip:/usr/lib64/python3.12/:/usr/lib64/python3.12/lib-dynload:/usr/local/lib/python3.12/site-packages:/usr/lib64/python3.12/site-packages:/usr/lib/python3.12/site-packages
-LLM_NGPUS=-1
-LLAMA_CPP_CONFIG=llama_cpp_server_config.json
+LLAMA_ARG_MODEL="/models/mistral-7b-instruct-v0.2.Q4_K_S.gguf"
+LLAMA_ARG_ALIAS="default-model"
+LLAMA_ARG_N_GPU_LAYERS=-1
+LLAMA_ARG_THREADS=12
+LLAMA_ARG_BATCH=512
+# Modify following var when switching model
+LLAMA_ARG_CHAT_TEMPLATE="mistral-v3"
```

Comment:
are these compatible? v3 vs. our v0.2 model?

Reply:
Yes, the chat templates are actually identical in version 2 and 3.
The tokenizer has been slightly modified, but not in a way that would affect our use case.
https://github.com/mistralai/cookbook/blob/main/concept-deep-dive/tokenization/chat_templates.md